docs/indexing.md

```diff
@@ -56,24 +56,28 @@ SSH to a bibdata machine and start a tmux session
 
 as deploy user, in `/opt/marc_liberaton/current`
 
-`$ RAILS_ENV=production SET_URL=http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging-rebuild bin/rake liberate:full --silent >> /tmp/full_reindex_[YYYY-MM-DD].log 2>&1`
+`$ SET_URL=http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging-rebuild bin/rake liberate:full`
 
-This step takes about 21 hours
+Indexing jobs for each DumpFile in the dump will be run in the background. To watch the progress of the index, you can login and go to /sidekiq.
```

Comment:
"you can login" Login to what?

Reply:
bibdata. I'll clarify.

Reply:
@hectorcorrea I updated this is it better?